doc/index.rst

```diff
@@ -29,6 +30,7 @@ Welcome to the Cabal User Guide
    external-commands
    setup-commands
    file-format-changelog
+   version-control-fields
```

Comment:
This page's position better fits right after the two cabal description files as it strongly belong to them

Reply:
Thanks, I'd put it next to the "Field Syntax Reference" but have moved it as you suggest.